app/ExposureNotificationContext.tsx

```diff
@@ -35,7 +35,7 @@ const ExposureNotificationsProvider = ({
     const cb = (authorizationStatus: ENAuthorizationStatus) => {
       setAuthorizationStatus(authorizationStatus);
     };
-    ExposureNotifications.requestAuthorization(cb);
+    BTNativeModule.requestAuthorization(cb);
```

Comment:
This looks breaking for GPS? Should this be a switch on the flavor?

Reply:
so long as the function isnt called for the gps build it wont throw, as RN handles the subscription across the bridge at run time and we are instantiating the emitter on invocation of the function.  

Though perhaps thats not super clear by reading the function itself, so we could put a redundant !isGPS in here, but id really like to get away from all that checking and it would be ideal to have each tracing strategy not need to be away of each of the others.

Reply:
Ah, I see. If GPS also needs a step in the flow here, it might make sense to have this decide the request that's made, so that the provided function is only the relevant one. I'm a little wary of providing a function to components that they shouldn't call. 
This sounds good for now then as we don't know on the GPS side what that would look like

Reply:
> I'm a little wary of providing a function to components that they shouldn't call.

same here! I think we could take a pass at make this a little harder to get at for a random component on the tree.

like if we case on the app mounting conditionally include these there. it wouldnt be fool proof but it would be pretty easy to implement and youd have to go out of your way to pull this in.